notebooks/official/pytorch_image_classification_distributed_data_parallel_training_with_vertex_sdk/multi_node_ddp_gloo_vertex_training_with_custom_container.ipynb

```diff
@@ -0,0 +1,779 @@
+{
```

Comment:
<p>how to use create -&gt; how to create</p><p> </p><p>Vertex SDK n-&gt; Vertex AI SDK</p><p> </p><p>Vertex Tensorboard -&gt; Vertex AI TensorBoard</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>